### PR TITLE
Book and Lable Class

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
- Complete feature branch with reference to ***issue #3 on the Kanban board***

### CODE REVIEW

File: classes/lable.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(id:, title:, color:)
+    @id = id || Random.rand(1..1000)
```
Comment (youmari) [inline classes/lable.rb:6]: @Chimwemwe-127001 I think the id should be automatically given to the label without  giving the user the choice to choose

**Review by youmari — CHANGES_REQUESTED** [review]

File: classes/lable.rb (hunk 1 above)
Comment (Chimwemwe-127001) [reply classes/lable.rb:6]: Noted. Working on the changes now.

**Review by samwel-mongare — APPROVED** [review]
Hi @Chimwemwe-127001  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# status 🟢 approved
Hi @Chimwemwe-127001  you can merge it now good job

**Pull request merged**